spec/hbw/features/tasks/task_list_with_filter_bp_mock.yml

```diff
@@ -0,0 +1,375 @@
+                  delete_if:
+                    - variable: homsOrderDataZeroVariable
+                      condition: $var == 0
+                - name: homsOrderDataSomeText
+                  type: text
+                  label: Text
+                  css_class: col-xs-6 col-sm-4 col-md-3
+                  delete_if:
+                    - variable: homsOrderDataZeroVariable
+                      condition: $var == 0
+                - name: homsOrderDataSomeDisabledText
+                  type: text
+                  label: Disabled Text
+                  css_class: col-xs-6 col-sm-4 col-md-3
+                  disable_if:
+                    - variable: homsOrderDataZeroVariable
+                      condition: $var == 0
+            - name: group2
+              type: group
+              label: Hidden group
+              css_class: col-xs-12
+              fields: []
+              delete_if:
+                - variable: homsOrderDataZeroVariable
+                  condition: $var == 0
+
+  task/111112/deployed-form: *form_with_everything
+
+  # task variables
+  task/111111/variables:
+    - params: ""
+      response:
+        homsOrderCode:
+          type: "string"
+          value: "ORD-1"
+        homsOrderDataSelect:
+          type: "string"
+          value: null
+
+  task/111112/variables:
+    - params: ""
+      response:
+        homsOrderCode:
+          type: "string"
+          value: "ORD-2"
+        homsOrderDataSelect:
+          type: "string"
+          value: null
+
+  task/111111:
+    - params: ""
+      response: &default_task
+        id: "111111"
+        name: "Check test form"
+        assignee: "demo"
+        created: "2016-06-30T12:07:59.000+03:00"
+        due: null
+        followUp: null
+        delegationState: null
+        description: null
+        executionId: "222221"
+        owner: null
+        parentTaskId: null
+        priority: 50
+        processDefinitionId: "testProcess:469:222221"
+        processInstanceId: 222221
+        taskDefinitionKey: "usertask10"
+        caseExecutionId: null
+        caseInstanceId: null
+        caseDefinitionId: null
+        suspended: false
+        formKey: "test_form_with_required_select_lookup.yml"
+        tenantId: null
+
+post:
+  task:
+    - params: "active=true&assignee=demo&processVariables[][name]=homsOrderCode&processVariables[][operator]=like&processVariables[][value]=%"
+      response:
+        - <<: *default_task
+          id: "111111"
+          formKey: "test_form_with_disabled_buttons.yml"
+          processInstanceId: 222221
+          processDefinitionId: "testProcess:469:222221"
+          assignee: null
+        - <<: *default_task
+          id: "111112"
+          name: "Assigned task"
+          due: "2016-07-30T12:07:59.000+03:00"
+          priority: 25
+          processInstanceId: 222222
+          processDefinitionId: "testProcess:469:222222"
+        - <<: *default_task
+          id: "111111"
```

Comment:
two tasks with the same ids?

Reply:
It was copied from https://github.com/latera/homs/blob/14a88d5dfd26838b60af44fcc2eb5529c74abda2/spec/hbw/features/tasks/task_list_mock.yml#L294 

I fixed it but It didn't affect on test results